test/config-map.test.ts

```diff
@@ -317,5 +313,142 @@ test('can configure an immutable config map', () => {
 
   expect(cm.immutable).toBeTruthy();
   expect(spec.immutable).toBeTruthy();
+});
+
+test('computes checksum on data', () => {
+  // GIVEN
+  const chart = Testing.chart();
+
+  // WHEN
+  const cm = new ConfigMap(chart, 'my-config-map', {
+    data: {
+      key1: 'foo',
+      key2: 'bar',
+    },
+  });
+  const ns = new Namespace(chart, 'my-namespace', {});
+  cm.addChecksumTo(ns);
```

Comment:
Please add documentation for this feature to https://github.com/cdk8s-team/cdk8s-plus/blob/k8s-30/main/docs/plus/config-map.md